controller/src/main/java/io/pravega/controller/server/eventProcessor/requesthandlers/StreamRequestHandler.java

```diff
@@ -73,15 +59,8 @@ public boolean toPostpone(ControllerEvent event, long pickupTime, Throwable exce
     }
 
     @Override
-    public CompletableFuture<Void> processAbortTxnRequest(AbortEvent abortEvent) {
-        return Futures.failedFuture(new RequestUnsupportedException(
-                "StreamRequestHandler: abort txn received on Stream Request Multiplexer"));
-    }
-
-    @Override
-    public CompletableFuture<Void> processCommitTxnRequest(CommitEvent commitEvent) {
-        return Futures.failedFuture(new RequestUnsupportedException(
-                "StreamRequestHandler: commit txn received on Stream Request Multiplexer"));
+    String getProcessorName() {
+        return this.getClass().getSimpleName();
```

Comment:
If this is what all your implementations look like, then just put this body into the base class' implementation and only override it if you have something different to say.